docs/howto/openstack/cinder/retype-volumes.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Changing the type of a volume ("retyping") is an offline operation that requires detaching the volume from its server, and setting its new type.
+---
+# Changing a volume's type
+
+You may occasionally need to change the type of a volume.
+This may be due to a volume type being phased out on {{brand}}'s part (necessitating data migration), or you might want to enable or disable [encryption](encrypted-volumes.md) on a volume.
+
+{{page.meta.description}}
+The resulting down time may be quite substantial, particularly for large volumes.
+As such, when you need to retype volumes, you should plan ahead well in advance.
+
+## Prerequisites
+
+In order to retype volumes, you must use the OpenStack CLI.
+Make sure you have [enabled it](../../getting-started/enable-openstack-cli.md).
+
+If you are about to retype a large volume, or one that holds data associated with a critical service, you may be interested in an estimate of how long the retype operation will take.
+In that case, please file a support request with our [{{support}}](https://{{support_domain}}).
+
+## Checking the volume's state
+
+Assume you have a volume named `testvol` that is currently attached to a server named `testsrv`:
+
+```console
+$ openstack volume list --long
++----------------+---------+--------+------+---------+----------+----------------+------------+
+| ID             | Name    | Status | Size | Type    | Bootable | Attached to    | Properties |
++----------------+---------+--------+------+---------+----------+----------------+------------+
+| e233e7f3-f33b- | testvol | in-use |   50 | default | false    | Attached to    |            |
+| 4d7a-8f5b-785b |         |        |      |         |          | testsrv on     |            |
+| 34f670bf       |         |        |      |         |          | /dev/vdb       |            |
++----------------+---------+--------+------+---------+----------+----------------+------------+
+```
+
+In this example, this volume's status is `in-use`, meaning it is currently attached to a server, and the volume's type is `default`.
```

Comment:
I would not use the possessive form --- not for the volume, at least. So, I would write something like this: "In this example, the volume status is `in-use`, meaning it is currently attached to a server and that the volume type is `default`".

Reply:
I think it's a toss-up, but yeah we can go without the possessive.